K2Bridge/Factories/BucketFactory.cs

```diff
@@ -119,5 +119,29 @@ public static RangeBucket CreateRangeBucket(string primaryKey, DataRow row, ILog
 
             return rb;
         }
+
+        /// <summary>
+        /// Create a new <see cref="HistogramBucket" from a given <see cref="DataRow"/>/>.
+        /// </summary>
+        /// <param name="row">The row to be transformed to bucket.</param>
+        /// <returns>A new HistogramBucket.</returns>
+        public static HistogramBucket CreateHistogramBucket(string primaryKey, DataRow row, ILogger logger)
+        {
+            Ensure.IsNotNull(row, nameof(row));
+
+            var key = row[0];
+            var count = row[BucketColumnNames.Count];
+            var keyed = row.Table.Columns[0].ToString().Split('%')[1];
```

Comment:
TODO - when date range is merged, use the constant for %

Reply:
*merged*